numpy/core/src/multiarray/einsum.c.src

```diff
@@ -37,6 +37,17 @@
 #define EINSUM_USE_SSE2 0
 #endif
 
+#ifdef NPY_HAVE_NEON
```

Comment:
Use the following guards instead of CPU feature definitions.
```C
#if NPY_SIMD
// a valid guard for all data types except for float64 
#endif

#if NPY_SIMD_F64
// a special guard for float64, to add combitablty with arm7/neon
#endif
````